### DISCUSSION TITLE
Add extra attributes to temperature cluster

### DISCUSSION THREAD

**Discussion by talhamir49** [Koenkk/zigbee2mqtt#8704] [Devices]
Hi Everyone.

I'm trying to add new attributes to the temperature cluster like measuredvalues2 3 and  4. successfully added on end-node but am unable to add it in ZIgbee2mqtt. can someone please guide me?

**Comment by talhamir49** [comment]
@Koenkk  I added the additional attributes in the cluster file and make attributes reportable by adding them in reporting file but when I access it in my converter file to set reporting setting it gives an error "UNSUPPORTED_ATTRIBUTE". Can you please tell me if there is any other file to add attributes?

**Reply by Koenkk** [reply]
`UNSUPPORTED_ATTRIBUTE` means your device does not support it.

**Reply by talhamir49** [reply]
@Koenkk  Thanks for the response. problem solved.
